pyteal/ast/abstractvar.py

```diff
@@ -38,3 +38,32 @@ def storage_type(self) -> TealType:
 
 
 AbstractVar.__module__ = "pyteal"
+
+
+def alloc_abstract_var(stack_type: TealType) -> AbstractVar:
+    """Allocate abstract var over stack, or over scratch.
+
+    This unexported function takes a TealType as value type representation over stack (or scratch),
+    and generates an AbstractVar instance.
+    It infers the proto currently being used in context of subroutine evaluation,
+    and swap to FrameVar to save the use of scratch slots.
+
+    Arg:
+        stack_type: An TealType that represents stack type.
```

Comment:
@ahangsu Minor verbiage suggestion.  Alternatively, changing from _An_ to _A_ works too.

```suggestion
        stack_type: TealType that represents stack type.
```